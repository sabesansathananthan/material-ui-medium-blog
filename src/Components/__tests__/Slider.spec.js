import React from "react";
import { shallow } from "enzyme";
import Slider from "../Slider";
import toJson from "enzyme-to-json";
import axios from "axios";

describe("Slider component", () => {
  let wrapper;
  let getSpy;
  beforeAll(() => {
    getSpy = jest.spyOn(axios, "get");
    wrapper = shallow(<Slider />);
  });

  it("should render without errors", () => {
    expect(wrapper).toBeDefined();
  });

  it("should render correctly - snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("should fetch a list of posts", () => {
    expect(getSpy).toBeCalled();
  });
});
