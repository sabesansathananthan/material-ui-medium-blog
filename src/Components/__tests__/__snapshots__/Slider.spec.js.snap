// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider component should render correctly - snapshot 1`] = `""`;
